torch/ao/quantization/fx/quantization_patterns.py

```diff
@@ -523,7 +409,28 @@ def modified_load_arg(n: Node):
                     self.relu_node)
             else:
                 return quantized_graph.node_copy(node, load_arg(quantized=torch.float))
+        else:
+            if self.num_tensor_args == 2:
+                # make sure both inputs are quantized to act_dtype
+                load_arg(quantized={0: act_dtype, 1: act_dtype})(self.binary_op_node.args)
+            args = load_arg(quantized=torch.float)(self.binary_op_node.args)
+            kwargs = load_arg(quantized=torch.float)(self.binary_op_node.kwargs)
+            op_out = quantized_graph.node_copy(self.binary_op_node, load_arg(quantized=torch.float))
+
+            def modified_load_arg(n: Node):
```

Comment:
I realize this was in the original code, but would be great to remove this in the future to increase readability

Reply:
yeah 90% of code here will be removed after the refactor, including this and the overall load_arg stuff